python/lsst/daf/butler/core/storageClass.py

```diff
@@ -28,11 +28,51 @@
 
 class StorageClassMeta(type):
 
+    # Caches of imported code objects.
```

Comment:
Do we still actually need the metaclass and its registry of subclasses? If not, we should remove it, and move its methods to regular `StorageClass` class methods, to reduce complexity.

Reply:
You need a metaclass to support lazy loading of class attributes. The `subclasses` stuff is not used at the moment but I have wondered if it might be useful (will we ever need to locate a storage class that hasn't been put in the storage class registry). Can remove it easily enough once the code settles down.

Reply:
oh, and this is metaclass lazy loading so I was wondering about how to document a metaclass at all.